lib/elixir_internal_certificate_worker/scraper/helpers/job_queue_helper.ex

```diff
@@ -0,0 +1,9 @@
+defmodule ElixirInternalCertificateWorker.Scraper.JobQueueHelper do
+  alias ElixirInternalCertificateWorker.Scraper.UserSearchWorker
+
+  def create_searching_queue(uploaded_keywords) do
```

Comment:
normally we name this kind of method as `enqueue_user_search_worker`, what do you think about that name 🤔